src/components/Navigation/Header.vue

```diff
@@ -12,78 +12,83 @@
       <v-img
         src="@/assets/fairsharing-logo.svg"
         height="70"
-        class="d-flex flex-grow-0"
         contain
       />
     </router-link>
-    <v-spacer />
-    <nav>
-      <ul
-        v-if="!$vuetify.breakpoint.sm && !$vuetify.breakpoint.xs"
-        class="d-flex flex-row align-center flex-wrap"
-      >
-        <li
-          v-for="(item, itemIndex) in links"
-          :key="'navBarTopMenuItem_' + itemIndex"
+    <div
+      id="custom-width"
```

Comment:
Use a class rather than an ID plz.
ID should be meaningful and describe what the item is about semantically speaking, not a means to access CSS properties :)